### DISCUSSION TITLE
init Neo4j Client failed

### DISCUSSION THREAD

**Discussion by Sun-Terry** [OpenSPG/openspg#387] [Q&A]
![image](https://github.com/user-attachments/assets/f5d8938f-a214-498b-a6dc-00bab1971e3d)

**Comment by laggun** [comment]
我也遇到了相同的问题,请问你成功解决了吗

**Reply by Sun-Terry** [reply]
Yes, I started this client manually in the Docker UI

**Comment by bader1000** [comment]
https://github.com/OpenSPG/openspg/discussions/387#discussioncomment-11299351
